.env.example

```diff
@@ -72,6 +72,12 @@ OAUTH_KEYCLOAK_VERSION=
 # If true, sign ins and sign ups will only be possible through the OAuth providers configured above
 SSO_ONLY_MODE=
 
+# image exif cleaning options
+# available value: none, sanitize, scrub
+EXIF_CLEAN_MODE=sanitize
```

Comment:
keep in mind, to also set some defaults. To avoid regression during updating for server owners who do not yet have this option in the config.

Reply:
I'll see what I can do